src/server/util/Util.js

```diff
@@ -18,19 +18,7 @@ function post(req, callback) {
 
 function get(req, callback) {
     if(req.method === 'GET') {
-        let body = ''
-        
-        req.on('data', chunk => {
-            body += chunk.toString()
-        })
-
-        req.on('end', () => {
```

Comment:
I think this part can catch error if something is not handled in callback, so I think you can restore this.